src/components/Carts/checkout.jsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import {
+    Modal,
+    ModalHeader,
+    ModalBody,
+    Button,
+    Row,
+    Col,
+    Label
+} from "reactstrap";
+import { Control, LocalForm, Errors } from "react-redux-form";
+
+const Checkout = ({showModal , handleShow} ) => {
+    
+
+    const required = val => val && val.length;
+    const maxLength = len => val => !val || val.length <= len;
+    const minLength = len => val => val && val.length >= len;
+
+    const handleSubmit = (values) => {
+        // this.props.addComment(this.props.dishId, values.rating, values.author, values.comment);
+    }
```

Comment:
What is this for?
Also FYI `this` is not available inside functions

Reply:
I will remove this I was just forgot to remove this